homeassistant/components/unifiprotect/binary_sensor.py

```diff
@@ -216,33 +219,43 @@ def _async_update_device_from_protect(self) -> None:
 class ProtectDiskBinarySensor(ProtectNVREntity, BinarySensorEntity):
     """A UniFi Protect NVR Disk Binary Sensor."""
 
+    _disk: UOSDisk
```

Comment:
This has the same type in the signature. Maybe we can remove the type info here?